### PR TITLE
feat: update font file and add meta tags

### LINKED ISSUE #379: Add open graph for better experience when sharing on social medias
- https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwww.excalidraw.com%2F
  `og:url`, `og:type`, `og:title`, `og:image`, `og:description`
- https://cards-dev.twitter.com/validator

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR updates the `FF_Virgil` font file (per discussion in #394) to use a smaller file and adds the meta tags, which should show image when sharing on social media sites like Facebook and Twitter. 

## Changes

- update font file and font file links 
- add og and ~twitter~ meta tags
- add og:image:width and og:image:height tags
- ~add PR template~

## Screenshots

The og image that was added:
og-image

## Checklist

- [x] tested locally
- [ ] updated the docs

Fixes #379

### CODE REVIEW

**Comment by jsjoeio** [comment]
Tested using both [Facebook's Sharing Debugger](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fexcalidraw-git-jsjoeio-fix-font-file.vjeux.now.sh%2F) and [Twitter's Card Validator](https://cards-dev.twitter.com/validator) using the Now preview URL but not getting the expected results. 

🤞🏼should change once merged into production. Happy to follow up and test and make changes if needed.

File: public/index.html [hunk 1]
```diff
@@ -8,11 +8,37 @@
...
+      property="og:description"
+      content="Excalidraw is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them."
+    />
+    <meta property="og:image" content="%PUBLIC_URL%/og-image.png" />
+
+    <!-- Twitter Card tags -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@Vjeux" />
+    <meta name="twitter:creator" content="@Vjeux" />
+    <meta name="twitter:image" content="%PUBLIC_URL%/og-image.png" />
```
Comment (lipis) [inline public/index.html]: could me merge with the property image..

```
<meta property="og:image" name="twitter:image"...
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+## Screenshots
+
+<!-- (prefer animated gif) -->
+
+## Checklist
+
+- [ ] tested locally
+- [ ] updated the docs
+
+Fixes #
```
Comment (lipis) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not part of this PR.. Maybe we should skip that.

**Comment by lipis** [comment]
Fixes #379

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (lipis) [reply .github/PULL_REQUEST_TEMPLATE.md]: To be honest, I don't really like these templates, if we see that is needed in the future we could reconsider.

Comment (jsjoeio) [reply .github/PULL_REQUEST_TEMPLATE.md]: Okay, I'll remove that

File: public/index.html (hunk 1 above)
Comment (jsjoeio) [reply public/index.html]: Sure!

**Comment by pshihn** [comment]
Recommend adding `og:image:width` and `og:image:height` and equivalent twitter ones as well. 

"Using these tags will specify the image dimensions to the crawler so that it can render the image immediately without having to asynchronously download and process it."
https://developers.facebook.com/docs/sharing/best-practices/

Also, back in the day FB did not like relative URLs for the `og:image` not sure if that's still the case.

**Comment by Fausto95** [comment]
Just checked the font, the rendering of ! is perfect now

**Comment by jsjoeio** [comment]
Thanks for the suggestion @pshihn! I'll add those in now

**Comment by jsjoeio** [comment]
Okay, I think it should be good now if someone wants to review it again.

Re: @pshihn 
> equivalent twitter ones as well

I had the Twitter meta tags in there but @lipis [suggested](https://github.com/excalidraw/excalidraw/pull/400#discussion_r367622223) opting for the `og:image` instead. 

From the quick research I did, I think it should be fine. 

> If an og:type, og:title and og:description exist in the markup but twitter:card is absent, then a summary card may be rendered.

[Source](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup)

**Comment by lipis** [comment]
Maybe also update the image in the readme with this one?

**Comment by Fausto95** [comment]
> Maybe also update the image in the readme with this one?
Why not?!

I'd center the "Sketch hand-drawn like diagrams" and 👌🏼

**Comment by jsjoeio** [comment]
> Maybe also update the image in the readme with this one?
Good idea, done.

> I'd center the "Sketch hand-drawn like diagrams"

Done! 
og-image

**Comment by Fausto95** [comment]
Let's ship this and see if the results of the preview cards change!

**Comment by Fausto95** [comment]
Capture d’écran 2020-01-16 à 21 35 26

@jsjoeio the image doesn't show up

**Comment by jsjoeio** [comment]
@Fausto95 you're right, i'm seeing the same thing

image

I'll see if I can figure it out. The frustrating part is you can't use the Twitter card validator on localhost (for obvious reasons). I saw someone once suggest using `ngrok` but I tried that and ran into a canvas security error. Open to suggestions if you have any

**Comment by jsjoeio** [comment]
I wonder if it has anything to do with it being a relative path... 

AH-HA - look what I found:

> Documentation have nothing about it, but relative URLs will not work, only full URL including scheme works

https://stackoverflow.com/a/9858694/3015595

**Comment by Fausto95** [comment]
I hope it does haha.
Let's try it out
